functions/go/set-default-labels/transformer/labels.go

```diff
@@ -0,0 +1,52 @@
+package transformer
+
+import (
+	"fmt"
+
+	"github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/set-labels/transformer"
+	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
+)
+
+const (
+	AppNameKey = "app.kubernetes.io/name"
+	BuiltInName = "builtin"
```

Comment:
It seems it doesn't need to be public.

Reply:
Sure. I am thinking of making this "builtin" name as a convention. I'll move it to the SDK if I encounter other cases that need a autogenerated namespace. (i'm almost sure we will have some coming soon)